beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -932,16 +926,27 @@ type singleIndexedVerificationFailureJson struct {
 
 type nodeSyncDetailsErrorJson struct {
 	apimiddleware.DefaultErrorJson
-	SyncDetails syncDetails `json:"sync_details"`
+	SyncDetails syncDetailsJson `json:"sync_details"`
 }
 
 type eventErrorJson struct {
 	StatusCode int    `json:"status_code"`
 	Message    string `json:"message"`
 }
 
-type syncDetails struct {
+// ---------------
```

Comment:
Added an additional section to more accurately reflect the purpose of these structs